openedx/features/assessment/helpers.py

```diff
@@ -31,7 +31,8 @@ def can_auto_score_ora(enrollment, course, block, index_chapter):
         return False
     response_submission = Submission.objects.filter(
         student_item__student_id=anonymous_user.anonymous_user_id,
-        student_item__item_id=block).first()
+        student_item__item_id=block
+    ).order_by('id').first()
```

Comment:
please add a comment that in case of multiple submissions we need to pick the lates one.